styles/global.css

```diff
@@ -82,3 +96,34 @@ footer a:hover {
   text-decoration: underline;
   font-weight: bold;
 }
+
+@media only screen and (max-width: 600px) {
+  .container-aside-layout {
+    grid-template-columns: 1fr;
+    grid-template-rows: 100px 1fr;
+    grid-template-areas:
+      "header"
+      "main";
+  }
+
+  .sidebar {
+    display: none;
+  }
+
+  .nav-search {
```

Comment:
Everything looks good, one minor suggestion is that you could have done like this
`.nav-search, .nav-pills{
  display : none;
}`

Reply:
Resolved